APCAppCore/APCAppCore/UI/TabBarControllers/Profile/APCProfileViewController.m

```diff
@@ -725,8 +723,7 @@ - (NSArray *)prepareContent
             [rowItems addObject:row];
         }
         
-        if (self.user.sharedOptionSelection != [NSNumber numberWithInteger:SBBConsentShareScopeNone])
-        {
+        if (self.user.sharedOptionSelection != [NSNumber numberWithInteger:SBBConsentShareScopeNone]) {
```

Comment:
Indeed. I have this part commented out on my end because the `SBB...` part is defined in the BridgeSDK. This should be rewritten and probably get its own define/enum.